examples/cpp/content_filter/CLIParser.hpp

```diff
@@ -19,8 +19,8 @@
 
 #include <fastdds/dds/log/Log.hpp>
 
-#ifndef _FASTDDS_CONTENT_FILTER_CLI_PARSER_HPP_
-#define _FASTDDS_CONTENT_FILTER_CLI_PARSER_HPP_
+#ifndef FASTDDS_EXAMPLES_CPP_CONTENT_FILTER__CLI_PARSER_HPP
```

Comment:
NIT we usually name the file without spaces, as the name of the file
```suggestion
#ifndef FASTDDS_EXAMPLES_CPP_CONTENT_FILTER__CLIPARSER_HPP
```